test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -1296,4 +1297,26 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			assert.strictEqual( content, originalContent, 'The restored post content is not correct' );
 		} );
 	} );
+
+	describe( 'Settings @parallel', function () {
+		step( 'Can log in', async function () {
+			const loginFlow = new LoginFlow( driver, gutenbergUser );
+			await loginFlow.loginAndStartNewPost( null, true );
+		} );
+
+		step( 'Line height setting is available', async function () {
```

Comment:
I think instead of creating a separate `Settings` section, it would be better to have this under the [`Edit a post`](https://github.com/Automattic/wp-calypso/blob/57e257d9482c7b0c91b2f1033f2700dcbafa1587/test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js#L861) one. Sth like:

```js
describe( 'Edit a Post: @parallel', function () {
    describe( 'Paragraph settings', function () {
        step( 'Line height is available', function () {
                // ...
        } );
    } );
} );
```

What do you think?

Reply:
That makes sense! I'll change 👍🏻